test/cpp/jit/test_subgraph_matcher.h

```diff
@@ -361,6 +361,80 @@ graph(%x, %y):
   AT_ASSERT(findPatternMatches(pattern1, graph).size() == 0);
 }
 
+void testAttributes() {
+  Graph graph;
+  script::parseIR(
+      R"IR(
+graph(%0):
+  %a = a::a[isattr=[1,2]](%0)
```

Comment:
Looks mostly good. Question - for the case we are interested in, we will be matching only particular attribute [4th arg of dequantize_linear which is static], rest all are dynamic. How will the pattern look like for that case ?

Reply:
Good question, I don't have a definite answer to that: we could either 1) relax the check a little and verify that only attributes that are present match, or 2) introduce wildcard support for attributes (it's more sophisticated because we need to come up with a sufficiently general and yet not too complicated to use design here).

But for our use case, do you expect *attributes* to be dynamic? I though we would just have one argument that would be const - for that we would have a node in the pattern with the attribute, and this solution should work there - other arguments would just match as they did before (without any attributes). Am I missing something?